frameworks/person-escort-record/README.md

```diff
@@ -0,0 +1,50 @@
+# Person Escort Record framework
+
+This framework contains the sections, steps and questions that need to be completed to produce
+a Person Escort Record (PER) when moving somebody using the Book a secure move service.
+
+The framework contains a set of YAML defintions that will be used by the [Book a secure move API](https://github.com/ministryofjustice/hmpps-book-secure-move-api)
+and [client applications](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend) of the API to generate validation and form journeys.
+
+## Manifest files
+
+Each manifest file contains a section of the PER. Each manifest contains definitions for the steps and the questions that are required to be answered within a tree-like structure.
+
+Manifest files live in `./frameworks/[framework-name]/manifests/`.
+
+### Manifest options
+
+- `name` **(required)** - name of the section. Used for the step heading caption.
+- `steps` - a list of steps for this section. See [steps documentation](#steps) for options.
+
+## Steps
+
+Steps define a form page within a particular section. A section can be made up of multiple steps.
+
+### Step options
+
+- `name` **(required)** - the name of the step. Used for the page heading.
+- `slug` **(required)** - the URL section for this step
+- `questions` **(required)** - list of questions to ask on this step and in what order. Each item should match the filename of the question. See [questions documentation](#questions) for question options.
+- `next` - can be used to override the following step or to define complex step based logic. Value should match the `slug` property.
+
+## Questions
+
+Questions define the way in which to gather a piece of information. Questions can be nested using followup questions on a parent
+question and can contain dependency logic to define whether a question should only be answered if another question is answered
+in a particular way.
+
+Question files live in `./frameworks/[framework-name]/questions/` in a flattened structure.
+
+### Question options
+
+- `question_type` **(required)** - the type of question input. Current supported types: `radio`, `checkbox`, `text`, `textarea`
+- `question` **(required)** - the full text of the question, usually including a question mark, that should be displayed in forms and summary tables.
+- `hint` - hint text to display after the question, usually for advice on how to format an answer
+- `options` - for question types that require answers to be within a particular set of items. Usually for radios and checkboxes
+  - `label` **(required)** - text displayed on the option label
+  - `value` - value submitted to the server, defaults to value of `label`
+  - `followup` - list of questions to conditionally display if this question is answered with this value. This will add this question as a dependent question dynamically to any followup questions and include them in the step where this question is defined.
```

Comment:
Maybe `followup` might be more suited at a higher level not under options 🤔

Reply:
Or maybe we want to avoid boolean questions and stick to `radio` so yes/no will be defined here instead?

Reply:
>Maybe followup might be more suited at a higher level not under options 🤔

I think the benefit of nesting within the options allows us not to need to duplicate which value a followup is based on. 

>Or maybe we want to avoid boolean questions and stick to radio so yes/no will be defined here instead?

Yep, I think if we go with this approach and avoid having to create a unique type for "boolean" or "boolean_with_comments" it will allow us to use this approach consistently to display nested, followup questions.